src/Elmish.WPF/WpfProgram.fs

```diff
@@ -155,6 +158,16 @@ module WpfProgram =
     { program with LoggerFactory = loggerFactory }
 
 
+  /// Calls the specified function for unhandled exceptions in the Elmish
+  /// dispatch loop (e.g. in commands or the update function). This essentially
+  /// delegates to Elmish's Program.withErrorHandler. The first (string)
+  /// argument of onError is a message from Elmish describing the context of the
+  /// exception. Note that this may contain a rendered message case with all
+  /// data ("%A" formatting).
```

Comment:
One last thing.

Let's have this documentation say something about the relationship between this function and `WpfProgram.withLogger`.  Something about how the Emlish errors are also send to the logger, so providing both a logger and an Elmish error hander will result in being informed of this occurrence twice.

Reply:
Done